<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrearClaseGrupal</class>
 <widget class="QWidget" name="CrearClaseGrupal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Crear Clase Grupal</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelTitulo">
     <property name="styleSheet">
      <string notr="true">font-size:18pt; font-weight:bold;</string>
     </property>
     <property name="text">
      <string>Crear Nueva Clase Grupal</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelNombreClase">
     <property name="text">
      <string>Nombre de la Clase:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditNombreClase">
     <property name="placeholderText">
      <string>Introduce el nombre de la clase</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelTipoClase">
     <property name="text">
      <string>Tipo de Clase:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBoxTipoClase">
     <property name="placeholderText">
      <string>Selecciona tipo de clase</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnAñadirHorario">
     <property name="text">
      <string>Añadir Horario</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listaHorarios"/>
   </item>
   <item>
    <widget class="QPushButton" name="btnEliminarHorario">
     <property name="text">
      <string>Eliminar Horario Seleccionado</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelSociosApuntados">
     <property name="text">
      <string>Socios Apuntados:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listaSociosApuntados"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutBotones">
     <item>
      <widget class="QPushButton" name="btnGuardar">
       <property name="text">
        <string>Guardar Clase</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancelar">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
